modules/ROOT/pages/configuring-jmx-connection.adoc

```diff
@@ -13,19 +13,122 @@
+You can also map the administrator role for Single sign-on authentication with no user registry or the user that is not configured in the  user registry.
+Use the xref:reference:feature:appSecurity-1.0/examples.adoc#_configure_a_basic_user_registry_with_quickstart_security[quickStartSecurity] element for a single user mapping.
+Here is an example that shows the minimal required configuration:
+
+----
+ <featureManager>
+     <feature>restConnector-2.0</feature>
+ </featureManager>
+ <quickStartSecurity userName="bob" userPassword="bobpassword" />
+ <keyStore id="defaultKeyStore" password="keystorePassword"/>
+----
+
+Or use the basic registry for administrative role mapping.
+Here is an example of the basic registry that gives the user "bob" or the group "group1" administrator role:
+
+----
+ <basicRegistry>
+     <user name="bob" password="bobpassword"/>
+     <user name="joe" password="joepassword"/>
+     <group name="group1" ...>
+     </group>
+ </basicRegistry>
+
+ <administrator-role>
+     <user>bob</user>
+     <group>group1</group>
+ </administrator-role>
+----
+
+Or use the LDAP registry for administrator role mapping by adding the feature:ldapRegistry[LDAP User Registry feature] to your `server.xml` file.
+Here is an example of the LDAP registry that gives the user "bob" administrator role.
+
+----
+ <ldapRegistry id="basic" host="" port="">
+     <tds.properties ... />
+ </ldapRegistry>
+
+ <administrator-role>
+     <user>cn=bob,o=ibm,c=us</user>
+ </administrator-role>
+----
+
+Map the administrator role to an SSO user or group that does not exist in the configured user registry. First, configure SSO features, such as `samlWeb`, `openidConnectClient`, `socialLogin`, or `mpJwt`.
+
+Liberty security authorization is access-id based. An access-id takes following formats:
+
+access-id for a user:
+
+----
+ user:/<security realm name>/<user name>
+----
+
+access-id for a group:
+
+----
+ group:/<security realm name>/<group name>
+----
+
+For example, if a user is authenticated as `tom@example.com`, and the SSO server or issuer is identified as `https://idp.example.com`, the access-id for this user in Liberty issuer is: `user:https://idp.example.com/tom@example.com`.
+Assign an administrative role to a user authenticated by the SSO server. Map the role to the user's access-id instead of the user name. The following example shows how to assign an administrative role to the user `tom@example.com`, authenticated by the SSO server `https://idp.example.com`.
+
+----
+ <administrator-role>
+     <user-access-id> user:https://idp.example.com/tom@example.com</user-access-id>
+ </administrator-role>
+----
+
+Similarly, to map an administrative role to a group from an SSO server, you map the role to the group access-id. For example:
+
+----
+ <administrator-role>
+     <group-access-id>group:https://idp.example.com/ManagerGroups</group-access-id>
+ </administrator-role>
+----
+
```

Comment:
need to work on how these examples are introduced.  Don't use sentence fragments that end with the example